tigerpath/majors_and_certificates/majors/POL_2018.json

```
@@ -387,16 +647,16 @@
               ]
             },
             {
-              "name": "Track Departmentals",
+              "name": "Departmentals",
               "max_counted": 1,
               "min_needed": 4,
-              "explanation": "In addition to the prerequisite, students must complete four courses among those listed below along with six other departmental courses. Of the following six courses, students must take four with at least one being in Quantitative Analysis as well as at least one from the Game Theory and Applications category.",
+              "explanation": "In addition to the prerequisite, students must complete four courses among those listed below in either quantitative or analytical methods, along with six other departmental courses. Of the following six courses, students must take four with at least one being in Quantitative Analysis as well as at least one from the Game Theory and Applications category.",
               "req_list": [
                 {
                   "name": "Quantitative Analysis",
-                  "max_counted": 3,
+                  "max_counted": null,

```

```suggestion
                  "max_counted": 3,
```
It's redundant here, but will be robust in case more courses get added like with the Game Theory list (see below).